docs: add vulnerability_report.md

joshuajames-smith opened the pull request:
## Description

Template documentation addition of vulnerability_report.md which improves the gliff.ai GitHub Community Health Files.

*(note: file name uses _ instead of - to meet GitHub's requested format which also ensures consistency across the template documentation)

joshuajames-smith committed:
docs: add vulnerability_report.md

Template documentation addition of vulnerability_report.md which improves the gliff.ai GitHub Community Health Files.

*(note: file name uses _ instead of - to meet GitHub's requested format which also ensures consistency across the template documentation)

joshuajames-smith committed:
docs: change duplication warning

Request user checks for both open and previously closed issues on issue topic.

ChasNelson1990 reviewed (commented):
See comment on moving this to email

joshuajames-smith committed:
docs: delete vulnerability_report.md

Relates #1 

Delete vulnerability_report.md as team have decided not to publicly reveal security vulnerabilities, this will instead be handle via email for privacy and security.
